nupic/research/frameworks/vernon/handlers.py

```diff
@@ -1059,14 +1056,277 @@ def get_execution_order(cls):
         return eo
 
 
+class MetaContinualLearningExperiment(SupervisedExperiment):
+
```

Comment:
We need a docstring here. It should include the params used in `setup_experiment` as well as the distinctions between the meta-train/test phases and the fast and slow params. I'd like to suggest the following, reformatted as appropriate:

Generally:
* meta-training => teaching your model to learn efficiently from minimal examples
* meta-testing => testing how well your model can learn from minimal examples

Specifically:
* meta-train training = the inner loop training step(s)
* meta-train testing = the outer loop training step(s)
* meta-test training = training over most of a held-out validation dataset
* meta-test testing = testing over the rest of the held-out validation dataset

Params:
* fast-params = the parameters updated during meta-train training
* slow-params = the parameters updated during meta-train testing

Reply:
I added some documentation, and a TODO since there are some config args that Lucas added which I need to read up on before I can add to the docs.

Reply:
Thanks for these suggestions, I added in the docstring about the general meta-continual learning setup, but left config params to be described in `setup_experiment`, just as the parent class does.